plugins/lookup/hashi_vault.py

```diff
@@ -428,6 +440,22 @@ def auth_aws_iam_login(self):
             Display().warning("HVAC should be updated to version 0.9.3 or higher. Deprecated method 'auth_aws_iam' will be used.")
             self.client.auth_aws_iam(**params)
 
+    def auth_jwt(self, **kwargs):
+        role_id = kwargs.get('role_id', os.environ.get('VAULT_ROLE_ID', None))
```

Comment:
```suggestion
        role_id = self.get_option('role_id')
```
The code already handles environment variables.

Reply:
Method rewritten now. Resolving conversation.